### PR TITLE
Initial 0.1 version

### PR DESCRIPTION
**Pull request by RogerJung**

### CODE REVIEW

**Comment by evshary** [comment]
@RogerJung I think we should ignore the `build`, `log`, `install` folder and also some Python files like `__pycache__` or `*.pyc`.
I've already added the `.gitignore` to main branch

**Review by evshary — COMMENTED** [review]
Based on the discussion, we also need to add the README and create a PR for the autoware_carla_launch modification.
I'll give it a try later.

**Comment by evshary** [comment]
Let's start with 0.1 instead 1.0.

File: autoware_v2x/src/v2x_light/package.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>v2x_light</name>
+  <version>0.0.0</version>
```
Comment (evshary) [inline autoware_v2x/src/v2x_light/package.xml:5]: We can start with 0.1. Also, you can update description and maintainer meail.

Comment (evshary) [reply autoware_v2x/src/v2x_light/package.xml:5]: It's ok to use your own email.

File: traffic_manager/main.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+traffic_lights = None
+
+def main():
+
+    global traffic_lights
+
+    
+    # create a client in the Carla simulator
+    client = carla.Client('localhost', 2000)
```
Comment (evshary) [inline traffic_manager/main.py:58]: This can be an argument from CLI

File: autoware_v2x/src/v2x_light/resource/v2x_light
Comment (evshary) [inline autoware_v2x/src/v2x_light/resource/v2x_light:1]: Why do we have this empty file?

Comment (RogerJung) [reply autoware_v2x/src/v2x_light/resource/v2x_light:1]: It's the auto-generated marker file when I use "ros2 pkg create".

**Review by evshary — APPROVED** [review]
LGTM

**Pull request merged**